main/lib/plugins/CaliceGenericConverterPlugin.cc

```diff
@@ -191,25 +217,28 @@ namespace eudaq {
 	//  cout << "collection added." << endl;
       }
       col->parameters().setValue("DataDescription", dataDesc);
-      //add timestamp (EUDAQ!!)
+      //add timestamp (set by the Producer, is EUDAQ, not real timestamp!!)
       struct tm *tms = localtime(&timestamp);
       char tmc[256];
       strftime(tmc, 256, "%a, %d %b %Y %T %z", tms);
```

Comment:
Ok, just wanted to make sure that there is no problem because the syntax highlighting of github marked just these percent signs red. Had already fun with encoding problems when exchanging code between different os and editors.